src/ignitus-Common-Auth-Student-Prof/Components/createSignInPage.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withErrorBoundary } from '../../ignitus-Internals';
+import * as t from './constants';
+import '../Styles/style.scss';
+
+interface SignInConfig {
+  title: string;
+  studentRoute: string;
+  professorRoute: string;
+}
+
+function createSignInPage(config: SignInConfig): React.ComponentType {
```

Comment:
I think this would look better here, use Props it will look more generic , `SignInConfig` => component is for `signIn`
```
interface Props {
  title: string;
  studentRoute: string;
  professorRoute: string;
}
```
`function createSignInPage({ title,  studentRoute,  professorRoute }: Props): React.ComponentType { }`

Reply:
Sure, will update it to Props 👍 
I named it config to give a feel of component builder instead of a component.